packages/stats/gbstats/bayesian/tests.py

```diff
@@ -57,6 +68,20 @@ class GaussianBayesianConfig(BayesianConfig):
     epsilon: float = 1e-4
 
 
+@dataclass
+class EffectConfigFlatPrior(GaussianBayesianConfig):
+    prior_a: GaussianPrior = field(default_factory=GaussianFlatPrior)
+    prior_b: GaussianPrior = field(default_factory=GaussianFlatPrior)
+    prior_effect: GaussianFlatPrior = field(default_factory=GaussianFlatPrior)
+
+
+@dataclass
+class EffectConfigStandardPrior(GaussianBayesianConfig):
```

Comment:
These two should instead be one type, with BayesianConfig as the parent class e.g.

```
class EffectBayesianConfig(BayesianConfig):
    prior_a: GaussianPrior = field(default_factory=GaussianFlatPrior)
    prior_b: GaussianPrior = field(default_factory=GaussianFlatPrior)
    prior_effect: GaussianPrior = field(default_factory=GaussianFlatPrior)
```

You shouldn't need to create a second class to handle the config for the Effect Bayesian AB test. If you want a different config, you should instantiate `EffectBayesianConfig` with different arguments, e.g.: `EffectBayesianConfig(prior_effect=GaussianPrior(mean=0, variance=10, pseudo_n=100))`

Also update the class __init__ method to just take a dataclass of type `EffectBayesianConfig` since we are trying to stick to one config type per class type.

Reply:
Actually, shouldn't it just be:

```
class EffectBayesianConfig(BayesianConfig):
    prior_effect: GaussianPrior = field(default_factory=GaussianFlatPrior)
```

We don't use prior_a or prior_b in the EffectBayesianABTest class as far as I can tell.

Reply:
fixed